XamarinCommunityToolkitSample.iOS/Microsoft.Toolkit.Xamarin.Forms.Sample.iOS.csproj

```diff
@@ -0,0 +1,169 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
What's happening with this file? Looks like it has the Microsoft.Toolkit... name. So probably some wrong merge? :)

Reply:
@jfversluis strange, I've now deleted that file, also think I've fixed the the formatting